papers.html

```diff
@@ -137,6 +137,26 @@
     </nav>
     
 
+    <div class="container p-4">
+      <h1><a href="https://arxiv.org/abs/2411.02586" target="_blank" style="color: rgb(0,0,0)">Irregularly Sampled Time Series Interpolation for Detailed Binary Evolution Simulations</a></h1>
+      <p></p>
+      <h5>Srivastava et al. 2024</h5>
+      <p></p>
+
+      <p>
+        <button class="btn btn-secondary" type="button" data-toggle="collapse" data-target="#Srivastava20241105" aria-expanded="false" aria-controls="Srivastava20241105">
+          Details
+        </button>
+      </p>
+        <div class="collapse" id="Srivastava20241105">
+          <div class="card card-body">
+            <p><b>Authors: </b> Philipp M. Srivastava, Ugur Demir, Aggelos Katsaggelos, Vicky Kalogera, Elizabeth Teng, Tassos Fragos, Jeff J. Andrews, Simone S. Bavera, Max Briel, Seth Gossage, Konstantinos Kovlakas, Matthias U. Kruckow, Camille Liotine, Kyle A. Rocha, Meng Sun, Zepei Xing, Emmanouil Zapartas</p>
+            <p><b>Access: </b> <a href="https://arxiv.org/abs/2411.02586" target="_blank">arXiv:2411.02586</a> || <a href="https://ui.adsabs.harvard.edu/abs/arXiv:2411.02586" target="_blank">NASA ADS</a></p>
```

Comment:
The ADS link should become `https://ui.adsabs.harvard.edu/abs/2024arXiv241102586S` according to the ADS naming scheme.

Reply:
Neither is in ADS yet